lib/API/JSON/v1/Settings.php

```diff
@@ -61,6 +60,40 @@ function set($settings = []) {
     }
   }
 
+  private function onSettingsChange($old_settings, $new_settings) {
+    // Recalculate inactive subscribers
+    $old_inactivation_interval = $old_settings['deactivate_subscriber_after_inactive_days'];
+    $new_inactivation_interval = $new_settings['deactivate_subscriber_after_inactive_days'];
```

Comment:
Do we also need `isset` here?

Reply:
I don't think so, this setting should always be set. The other one depends on WooCommerce being active and also when it's disabled, the value is not in the database at all.